kernel/Tupfile.lua

```diff
@@ -1,9 +1,9 @@
 if tup.getconfig("NO_FASM") ~= "" then return end
 tup.rule("echo lang fix " .. ((tup.getconfig("LANG") == "") and "en" or tup.getconfig("LANG")) .. " > %o", {"lang.inc"})
-tup.rule("echo __REV = 0x`git rev-parse --short HEAD` > %o", {"rev.inc"})
-tup.rule({"bootbios.asm", extra_inputs = {"lang.inc", "rev.inc"}}, "fasm %f %o ", "bootbios.bin")

```

Comment:
I've lost, why do we remove the "rev.inc" dependency? Won't it cause the system to not make rev.inc in clean build?

Reply:
That's because the bootloader doesn't depend on `rev.inc` anymore, it is only used in the kernel.